ruby/pakyow/config.yaml

```diff
@@ -2,4 +2,4 @@ framework:
   website: pakyow.com
   version: 1.0
 
-command: bundle exec pakyow boot -e production
+command: bundle exec pakyow boot -e ludicrous
```

Comment:
what is `ludicrous` @bryanp ?

Reply:
It's a mode that configures an application for benchmarking (also a [reference to the movie "Spaceballs"](https://www.youtube.com/watch?v=oApAdwuqtn8) 😂)

Reply:
ah probably better to keep `production` mode. the idea is to reflect how can `pakyow` perform is real-world. wdyt ?

Reply:
Ah, good point. Reverted that commit: https://github.com/the-benchmarker/web-frameworks/pull/3216/commits/e839ea73050a5b2498801741bda63311ec515738